app/controllers/vacancies_controller.rb

```diff
@@ -4,7 +4,9 @@ class VacanciesController < ApplicationController
   before_action :set_landing_page_description, :set_map_display, only: %i[index]
 
   def index
-    params[:location] = params[:location_facet] if params[:location_facet]
+    # Set search parameters from pretty landing page params
+    search_params
```

Comment:
As this has side effects, can we call this method something more explicit? `set_params_from_pretty_landing_page_params` perhaps, then we don't even need the comment.

Reply:
And then it could be a before_action:

```ruby
before_action :set_params_from_pretty_landing_page_params, only: :index
```